wiki/Performance_troubleshooting/de.md

```diff
@@ -11,25 +7,25 @@ Aufgrund des Umfangs der Hardware, auf der osu! läuft, ist es leider sehr schwe
 - Läuft das Spiel im Fenster-Modus? Schalte um auf den Vollbildmodus.
-- Hast du deine Treiber aktualisiert (insbesondere die der GPU)? Versuche, das zu tun und schaue, ob sich die Dinge verbessern. Alternativ kannst du auch die Treiber zurücksetzen, wenn du sie vor kurzem aktualisiert hast.
-- Benutzt dein PC NVIDIA Optimus oder eine ähnliche hybride Grafiktechnologie? Wenn dem so ist, versuche, das Spiel auf jeder Grafikkarte laufen zu lassen, um die Leistung zu vergleichen. Es ist recht häufig, dass die integrierte Grafikkarte eine gleichmäßigere Leistung für Spiele wie osu! liefert. Suche nach "Grafikeinstellungen" in Windows und setze osu! zum Experimentieren entweder auf "Energiesparmodus" oder auf "Hohe Leistung".
+- Hast du deine Treiber aktualisiert (insbesondere die der GPU)? Versuche, das zu tun und schaue, ob sich die Dinge verbessern. Alternativ kannst du die Treiber auch zurücksetzen, wenn du sie erst vor kurzem aktualisiert hast.
+- Benutzt dein PC NVIDIA Optimus oder eine ähnliche hybride Grafiktechnologie? Wenn dem so ist, versuche, zu erzwingen, dass das Spiel einmal auf jeder GPU läuft, um die Leistung zu vergleichen. Es kommt recht häufig vor, dass die integrierte Grafikkarte eine gleichmäßigere Leistung für Spiele wie osu! liefert. Suche nach "Grafikeinstellungen" in Windows und setze osu! zum Experimentieren entweder auf "Energiesparmodus" oder auf "Hohe Leistung".
 
 ## Die Arten von "Lag"
 
 Es ist ziemlich schwierig, Unterstützung zu leisten, wenn jede Art von Problem unter der einzigen Bezeichnung “Lag” gruppiert wird. Fangen wir zunächst also an, einige Begriffe für spezifische Szenarien zu definieren.
 
 ### Stotterndes Audio
 
-Die eigentliche Audiospur der Beatmap spult im Takt zurück oder fällt aus. Grundsätzlich kannst du das am Audio hören, wenn das Spiel stockt. Beim Diagnostizieren oder Melden kann Folgendes helfen:
+Die eigentliche Audiospur der Beatmap spult zurück oder fällt aus. Kurz gesagt kannst du am Audio hören, ob das Spiel stockt. Beim Diagnostizieren oder Melden kann Folgendes helfen:
 
 - Prüfe, ob es auf allen verfügbaren Audiogeräten passiert. Wenn du einen USB-Kopfhörer benutzt, versuche stattdessen eine Verbindung über den analogen Ausgang deines PCs.
 - Prüfe, ob es hilft, den `Audio compatibility mode` in den Einstellungen zu aktivieren.
 
 ### Spike-Frame
 
-Während des Spielens friert alles lange genug ein, sodass das Spielerlebnis beeinträchtigt wird. Das kann mehr als einmal auftreten, nach dem Zufallsprinzip oder in gleichmäßigen Abständen. Beim Diagnostizieren oder Melden kann Folgendes helfen:
+Während des Spielens friert alles lange genug ein, um das Spielerlebnis zu beeinträchtigen. Das kann mehr als einmal auftreten, nach dem Zufallsprinzip oder in gleichmäßigen Abständen. Beim Diagnostizieren oder Melden kann Folgendes helfen:
```

Comment:
This would've seemed fine to me:
`Während des Spielens friert alles lange ein, sodass das Spielerlebnis beeinträchtigt wird.`

But the existence of the word `genug` (ca. `Alles friert lange genug ein`) implies that it not only freezes for a long time, but that the "freeze time" is long enough to interfere with the gameplay. That is why I use `, um ...` here.